portfolio/src/main/webapp/script.js

```diff
@@ -126,4 +126,29 @@ function randomSlide(randomButton) {
   const {totalSlides} = hideAllSlides(slider);
   const slideToShow = slider.children[Math.floor(Math.random() * totalSlides)];
   removeClass(slideToShow, "hidden");
+}
+
+/**
+ * Fetches comments from the server and adds it to the DOM.
+ */
+function getComments() {
+  const responsePromise = fetch('/data');
+  responsePromise.then(handleResponse);
```

Comment:
you can write all of this in a single line which then doesn't need a new variable:
```suggestion
  fetch('/data').then(handleResponse);
```

Reply:
Also what happens if for some reason, fetching fails?
See https://developer.mozilla.org/en-US/docs/Web/API/Response
it mentions that there is a `response.ok` flag to check whether the response was successful. Maybe you want to check this in `handleResponse` below. Note that the fetch promise is never rejected which means you don'
t have to add a `catch` case for it here.